Models/Models/BubbledChangeEventArgs.cs

```diff
@@ -73,27 +30,104 @@ public BubbledChangeEventArgs(IModelElement newElement)
         public EventArgs OriginalEventArgs { get; private set; }
 
         /// <summary>
-        /// Gets a value indicating whether the underlying change has been a elementary collection change
+        /// Gets the type of change that occured. This defines the type of OriginalEventArgs
+        /// and whether PropertyName is used.
         /// </summary>
-        public bool IsCollectionChangeEvent
```

Comment:
Removing these properties will introduce a breaking change. Thus, they should remain, though they can be implemented using the change type information.